src/dataset-builder/DatasetBuilder.ts

```diff
@@ -314,13 +314,14 @@ export const CohortSelector = ({
   return h(Fragment, [
     h(Selector<Cohort>, {
       headerAction: h(
-        Link,
+        ButtonOutline,
         {
+          style: { borderRadius: 0, fill: 'white', textTransform: 'none' },
           onClick: () => setCreatingCohort(true),
           'aria-label': 'Create new cohort',
```

Comment:
If you have a button with a label, you don't need (and probably shouldn't have) an aria label. Instead the button text becomes the aria label, and the unit test should use the button title to find its click target.